Formula/neatvi.rb

```diff
@@ -2,14 +2,14 @@ class Neatvi < Formula
   desc "Clone of ex/vi for editing bidirectional utf-8 text"
   homepage "https://repo.or.cz/neatvi.git"
   url "https://repo.or.cz/neatvi.git",
-      tag:      "08",
-      revision: "49e4029728927afb1a22864698be10cf445413aa"
+      tag:      "09",
+      revision: "a3b79df332e3c5804ae57c0348549ff35a69f262"
   head "https://repo.or.cz/neatvi.git"
 
   bottle do
     sha256 cellar: :any_skip_relocation, arm64_big_sur: "a49881c62f05798cf8016e0e6dfb3e00a368c86d338e9ddb332f3f0917e12706"
     sha256 cellar: :any_skip_relocation, big_sur:       "369536734eff4dea2a061a45b61060b9b91400599c2b5149cc56ec889b0c14a5"
-    sha256 cellar: :any_skip_relocation, catalina:      "c2c83bfb47a438d99fb7086a756292a4442dfb1d7eebd0314132814201cd944e"
+    sha256 cellar: :any_skip_relocation, catalina:      "c2c83bfb47a438d99fb7096a756292a4442dfb1d7eebd0314132814201cd944e"
```

Comment:
Why?

Reply:
I think Rui's script replaced 08 to 09 everywhere, something similar happened in a previous PR.

Reply:
exactly, the command replaced the 08 everywhere. it happened before. Correcting it now.